lib/bap_llvm/bap_llvm_ogre_coff.mli

```diff
@@ -0,0 +1,33 @@
+
+module Scheme : sig
```

Comment:
I think that we shouldn't provide `mli` files for the scheme, and schemes should be provided only as `ml` file. The justification, is that the `scheme` is an embedded DSL that describes data in our data-base. It is crucial to know the field names. For example, take the `section_header` value -- it is not usable, as you don't know how to reference the fields, you also don't know how to construct the values from other languages. On the other hand the definition: `Ogre.declare ~name:"section-header" (scheme name $ off $ size)` is self-describing.

So, provide each scheme as a separate `ml` file, that has only attributes and field definitions.